app/src/main/java/com/github/orelzion/spotifyclone/view/TrackFragment.kt

```diff
@@ -0,0 +1,40 @@
+package com.github.orelzion.spotifyclone.view
+
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import com.github.orelzion.spotifyclone.R
+import com.github.orelzion.spotifyclone.viewmodel.TracksViewModel
+
+class TrackFragment : Fragment(R.layout.fragment_track_item) {
+
+
+    /**
+     * Reference to the TextView that display track's details
+     */
+    private val selectedTrackFragment: TextView by lazy { requireView().findViewById(R.id.trackName) }
```

Comment:
please change the name of the var to a name that represents its purpose. Maybe `trackNameView`